subprojects/core/src/testFixtures/groovy/org/gradle/cache/internal/VersionSpecificCacheAndWrapperDistributionCleanupServiceFixture.groovy

```diff
@@ -26,7 +26,7 @@ import static org.gradle.cache.internal.WrapperDistributionCleanupAction.WRAPPER
 trait VersionSpecificCacheAndWrapperDistributionCleanupServiceFixture implements VersionSpecificCacheCleanupFixture {
 
     private static final BiAction<GradleVersion, File> DEFAULT_JAR_WRITER = { version, jarFile ->
-        jarFile << JarUtils.jarWithContents((GradleVersion.RESOURCE_NAME): "${GradleVersion.VERSION_NUMBER_PROPERTY}: ${version.version}")
+        jarFile << JarUtils.jarWithContents((GradleVersion.RESOURCE_NAME.substring(1)): "${GradleVersion.VERSION_NUMBER_PROPERTY}: ${version.version}")
```

Comment:
Why two different ways of removing the slash?

Reply:
Just to make sure the production code actually does the right thing.